typespec-extension/src/emitter.ts

```diff
@@ -42,6 +42,7 @@ export interface EmitterOptions {
   "polling"?: any;
 
   "arm"?: boolean;
```

Comment:
Not related to this PR.

`arm` is actually not a user settable option, right?
Then it better not be here at all.

If you need set it to Java, just do e.g. `(options as any).arm = codeModel.arm`